svcomp-benchmark/README.md

```diff
@@ -0,0 +1,8 @@
+The [SV-benchmarks](https://github.com/sosy-lab/sv-benchmarks/tree/master/c) consist of a large number of C programs that are used as verification tasks during the [SV-COMP](http://sv-comp.sosy-lab.org/2016/benchmarks.php). We analyzed 1346 programs classified as correct from the benchmarks, and ran them after compiling with [RV-Match](http://runtimeverification.com/match). For more information on how the programs were chosen, please see the [results](results/) directory. For information on how we compiled and ran the programs with RV-Match, please see the [scripts](scripts/) directory. 
```

Comment:
I think you should also mention the date here somewhere, at least in passing, because this data may make no sense 2-3 months or years from now.